lib/packwerk/cli.rb

```diff
@@ -122,16 +122,16 @@ def output_result(result)
       result.status
     end
 
-    sig { params(paths: T::Array[String], ignore_nested_packages: T::Boolean).returns(T::Array[String]) }
-    def fetch_files_to_process(paths, ignore_nested_packages)
-      files = FilesForProcessing.fetch(
-        paths: paths,
+    sig { params(relative_file_paths: T::Array[String], ignore_nested_packages: T::Boolean).returns(T::Array[String]) }
+    def fetch_files_to_process(relative_file_paths, ignore_nested_packages)
+      absolute_file_paths = FilesForProcessing.fetch(
```

Comment:
Those are not just `absolute_file_paths` they are the actual path for existing files. An absolute file path might exist or not. This code calculate the absolute path and return a file that exists. Maybe `files` or `found_files` is a better name.

Reply:
Ah -- so is the distinction here similar to how a `Pathname` can be in memory, but there may not be an underlying file at that path? So what you want is to be more explicit that it's not only a path (which may or may not exist) but a path that does exist?

Reply:
`absolute_files` makes sense to me.

Reply:
I pushed up this change